templates/etc/postfix/header_checks.j2

```diff
@@ -0,0 +1,5 @@
+# {{ ansible_managed }}
+{% for rule in postfix_header_checks %}
```

Comment:
{% for rule in postfix_header_checks | default([]) %}

Reply:
done in newest commit